### PR TITLE
Add 20005 port and extra cors headers for amplify

### PR DESCRIPTION
**Pull request by cosa65**

### CODE REVIEW

File: docker-compose.yaml [hunk 1]
```diff
@@ -15,6 +15,7 @@ services:
       # forward out edge port. set boto3's `endpoint_url`
       # to be http://localhost:4566 to use localstack for development
       - "4566:4566"
+      - "20005:4566"
```
Comment (aerlaut) [inline docker-compose.yaml:18]: Please correct me, the only 2 changes are this line

File: docker-compose.yaml [hunk 2]
```diff
@@ -25,6 +26,7 @@ services:
       - DOCKER_HOST=unix:///var/run/docker.sock
       - MAIN_CONTAINER_NAME='biomage-inframock-localstack'
       - LAMBDA_EXECUTOR=local
+      - EXTRA_CORS_ALLOWED_HEADERS=amz-sdk-request, amz-sdk-invocation-id
```
Comment (aerlaut) [inline docker-compose.yaml:29]: And this line to add CORS header for these requests?

File: docker-compose.yaml (hunk 1 above)
Comment (cosa65) [reply docker-compose.yaml:18]: yes, that is correct

Comment (cosa65) [reply docker-compose.yaml:18]: yes

File: docker-compose.yaml (hunk 2 above)
Comment (cosa65) [reply docker-compose.yaml:29]: yes

**Review by aerlaut — APPROVED** [review]
LGTM, tested locally

**Pull request merged**